docs/source/specs/openapi.json

```diff
@@ -3608,6 +3620,64 @@
                     }
                 }
             },
+            "TagRate": {
+                "required": [
+                    "metric"
```

Comment:
maybe I am missing something but I do not see `metric` under `TagRate`'s properties?

Reply:
This is a good question about the metric name, I will have to look into it a bit and come back with an answer.

Reply:
Update: Good catch @boaz0 I have this positioned in the wrong spot. I am going to move this over to WIP for now, and try to get those changes in today. I will ping you on slack when its ready for another review.